manifests/datastax_agent.pp

```diff
@@ -115,8 +115,7 @@
     $full_settings = {
       '' => $settings,
     }
-
-    validate_hash($full_settings)
+    assert_type(Hash, $full_settings)
```

Comment:
Isn't this really redundant since the variable set above and not user controlled?

Reply:
yeah right